### DISCUSSION TITLE
Can display do letterboxing?

### DISCUSSION THREAD

**Discussion by feklee** [ImageMagick/ImageMagick#2738] [General]
I want to `display` an image in a window of dimensions 2880×1620. If the aspect ratio of the image doesn't match, black bars should be added left/right or top/bottom. Currently I use as command line:

    display -resize 2880x1620 -immutable -geometry 2880x1620-0+0 image.jpg

However, that shrinks the window to the aspect ratio of the image, which is not what I want. The window should have the specified `geometry`.

**Comment by snibgo** [comment]
"display" doexsn't work on Windows so I can't test this, but instead of "-geometry 2880x1620-0+0" try "-gravity Center -background Black -extent 2880x1620".

**Reply by feklee** [reply]
There is no option `-extent` for `display`.
